### PR TITLE
Chilean chickpeas

### PR DESCRIPTION
**Pull request by matiasop**
Add chilean chickpeas recipe

### CODE REVIEW

File: src/chilean-chickpeas.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- ⏲️ Prep / Cook time: 30 - 40 m
+- 🍽️ Servings: 6
+
+## Ingredients
+
+- Dry chickpeas: 500 g
+- Sweet potato squash: 250 g
+- Onion: 1
+- Carrot: 1 cup
+- Pepper: 1 cup
```
Comment (hallzy) [inline src/chilean-chickpeas.md:13]:
```suggestion
- Grated carrot: 1 cup
- Grated pepper: 1 cup
```

Specifying that the carrot and pepper is grated right away makes it more clear on how you measure a carrot by volume.

Also, what type of pepper is this?

File: src/chilean-chickpeas.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- Oil: 2 tablespoon
+
+## Directions
+
+1. Let the chickpeas soak for at least 12 hours before cooking.
+2. Dice the onions.
+3. Grate the carrots.
+4. Cut the sweet potato squash into relatively big cubes.
+5. Chop the pepper and garlic into small pieces.
+5. On a pressure cooker, heat the oil, add the onion, carrot, celery and pepper. Cook for 5 minutes.
```
Comment (hallzy) [inline src/chilean-chickpeas.md:26]:
```suggestion
5. In a pressure cooker, heat the oil, add the onion, carrot, celery and pepper. Cook for 5 minutes.
```

File: src/chilean-chickpeas.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+6. Add the condiments (a tablespoon of salt, garlic, paprika).
+7. Add chickpeas and the sweet potato squash to the pressure cooker. Then fill with water until the preparation is covered.
```
Comment (hallzy) [inline src/chilean-chickpeas.md:28]:
```suggestion
7. Add chickpeas and the sweet potato squash to the pressure cooker. Then fill with water until everything is covered. 
```

File: src/chilean-chickpeas.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+2. Dice the onions.
+3. Grate the carrots.
+4. Cut the sweet potato squash into relatively big cubes.
+5. Chop the pepper and garlic into small pieces.
+5. On a pressure cooker, heat the oil, add the onion, carrot, celery and pepper. Cook for 5 minutes.
+6. Add the condiments (a tablespoon of salt, garlic, paprika).
+7. Add chickpeas and the sweet potato squash to the pressure cooker. Then fill with water until the preparation is covered. 
+8. Cook for 25 minutes in the pressure cooker.
+
+;tags: beans simple
```
Comment (hallzy) [inline src/chilean-chickpeas.md:31]:
```suggestion
;tags: chilean beans squash
```

Chilean I think would be good to keep.

File: src/chilean-chickpeas.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+- 🍽️ Servings: 6
+
+## Ingredients
+
+- Dry chickpeas: 500 g
+- Sweet potato squash: 250 g
+- Onion: 1
+- Carrot: 1 cup
+- Pepper: 1 cup
+- Celery: 1 cup
```
Comment (hallzy) [inline src/chilean-chickpeas.md:14]: Is the celery grated as well? Otherwise not sure how to measure celery in volume.

**Review by hallzy — CHANGES_REQUESTED** [review]

**Review by hallzy — APPROVED** [review]

**Review by DGKSK8LIFE — APPROVED** [review]

**Pull request closed without merging**